crates/sui-framework/tests/math_tests.move

```diff
@@ -18,6 +18,20 @@ module sui::math_tests {
         assert!(math::min(100, 10) == 10, 2);
         assert!(math::min(0, 0) == 0, 3);
     }
+    
+    #[test]
+    fun test_pow() {
+        assert!(nft::pow(1, 0) == 1, 0);
+        assert!(nft::pow(3, 1) == 3, 0);
+        assert!(nft::pow(2, 10) == 1024, 0);
+        assert!(nft::pow(10, 6) == 1000000, 0);
```

Comment:
Ah whoops, and one more issue.  The namespace here is not correct which some of the tests picked up.

```suggestion
        assert!(math::pow(1, 0) == 1, 0);
        assert!(math::pow(3, 1) == 3, 0);
        assert!(math::pow(2, 10) == 1024, 0);
        assert!(math::pow(10, 6) == 1000000, 0);
```

Reply:
🤦‍♂️ Sorry! That was a very silly mistake